### PR TITLE
Add French translations

### PR DESCRIPTION
**Pull request by Chxyz**
Hello,

I added the French translations to the package.

Best regards,
Chrys

### CODE REVIEW

File: Resources/Private/Translations/fr/Main.xlf [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff version="1.2"
+    xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file original="" product-name="Jonnitto.PrettyEmbedVideoPlatforms" source-language="en" target-language="en" datatype="plaintext">
```
Comment (jonnitto) [inline Resources/Private/Translations/fr/Main.xlf]:
```suggestion
    <file original="" product-name="Jonnitto.PrettyEmbedVideoPlatforms" source-language="en" target-language="fr" datatype="plaintext">
```

File: Resources/Private/Translations/fr/NodeTypes/Mixin/VideoID.xlf [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff version="1.2"
+    xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file original="" product-name="Jonnitto.PrettyEmbedVideoPlatforms" source-language="en" target-language="en" datatype="plaintext">
```
Comment (jonnitto) [inline Resources/Private/Translations/fr/NodeTypes/Mixin/VideoID.xlf]:
```suggestion
    <file original="" product-name="Jonnitto.PrettyEmbedVideoPlatforms" source-language="en" target-language="fr" datatype="plaintext">
```

File: Resources/Private/Translations/fr/NodeTypes/Mixin/Youtube/ClosedCaptions.xlf [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff version="1.2"
+    xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file original="" product-name="Jonnitto.PrettyEmbedVideoPlatforms" source-language="en" target-language="en" datatype="plaintext">
```
Comment (jonnitto) [inline Resources/Private/Translations/fr/NodeTypes/Mixin/Youtube/ClosedCaptions.xlf]:
```suggestion
    <file original="" product-name="Jonnitto.PrettyEmbedVideoPlatforms" source-language="en" target-language="fr" datatype="plaintext">
```

File: Resources/Private/Translations/fr/NodeTypes/Mixin/Youtube/ShowRelated.xlf [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff version="1.2"
+    xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file original="" product-name="Jonnitto.PrettyEmbedVideoPlatforms" source-language="en" target-language="en" datatype="plaintext">
```
Comment (jonnitto) [inline Resources/Private/Translations/fr/NodeTypes/Mixin/Youtube/ShowRelated.xlf]:
```suggestion
    <file original="" product-name="Jonnitto.PrettyEmbedVideoPlatforms" source-language="en" target-language="fr" datatype="plaintext">
```

**Review by jonnitto — CHANGES_REQUESTED** [review]
Hi @Chxyz 

Thank you for our PR! I really appreciate this. I've found some minor details to change

**Comment by Chxyz** [comment]
Hi,

Thank you for the review. My bad, forgot about that... I made the updates!

Best regards

**Review by jonnitto — APPROVED** [review]
No problem :-) Happens to the best ;-)

**Comment by jonnitto** [comment]
https://github.com/jonnitto/Jonnitto.PrettyEmbedVideoPlatforms/releases/tag/2.1.0

**Pull request merged**